push-down-test/run-tests.sh

```diff
@@ -0,0 +1,229 @@
+#!/usr/bin/env bash
+
+# check the pd and tikv binaries environment
+if [ -z $pd_bin ]; then
+  echo "PD server binary path not specified (pd_bin environment variable not set)"
+  exit 1
+fi
+
+if [ -z $tikv_bin ]; then
+  echo "TiKV server binary path not specified (tikv_bin environment variable not set)"
+  exit 1
+fi
+
+# these var is define by CI script
+# https://internal.pingcap.net/idc-jenkins/job/tikv_ghpr_integration-cop-push-down-test/configure
+tidb_src_dir=$tidb_src_dir
+tidb_bin=""
+pd_bin=$pd_bin
+tikv_bin=$tikv_bin
+# the hash of tidb_master_bin and pd_master_bin
+#tidb_master_sha=$tidb_master_sha
+#pd_master_sha=$pd_master_sha
+#tikv_cur_sha=$tikv_cur_sha
+
+readonly no_push_down_tidb_port="4005"
+readonly push_down_tidb_port="4006"
+readonly push_down_with_batch_tidb_port="4007"
+readonly tidb_host="127.0.0.1"
+readonly tidb_user="root"
+readonly log_level="warn"
+
+# This log location should be the same as which in CI script
+# https://internal.pingcap.net/idc-jenkins/job/tikv_ghpr_integration_pushdownfunc_test/configure
+readonly no_push_down_tidb_log_file="/tmp/copr_test/tidb_no_push_down.log"
+readonly push_down_no_batch_tidb_log_file="/tmp/copr_test/tidb_push_downno_batch.log"
+readonly push_down_with_batch_tidb_log_file="/tmp/copr_test/tidb_push_downwith_batch.log"
+readonly tikv_no_batch_log_file="/tmp/copr_test/tikv_no_batch.log"
+readonly tikv_with_batch_log_file="/tmp/copr_test/tikv_with_batch.log"
+readonly pd_no_batch_log_file="/tmp/copr_test/pd_no_batch.log"
+readonly pd_with_batch_log_file="/tmp/copr_test/pd_with_batch.log"
+
+readonly no_push_down_config_dir="./config/no_push_down"
+readonly push_down_no_batch_config_dir="./config/push_down_no_batch"
+readonly push_down_with_batch_config_dir="./config/push_down_with_batch"
+
+readonly push_down_test_bin="push_down_test_bin"
+
```

Comment:
`readonly push_down_func_test_bin="push_down_func_test_bin"
`

Reply:
It's better to keep the name same with the directory name, it may be used to test the pushed down executors.